Makefile

```diff
@@ -2,6 +2,13 @@
 build: index.js template.js components
 	@component build --dev
 
+build-browserify: $(SRC) node_modules
+	mkdir -p build
+	@browserify \
+	    --require query:component-query \
```

Comment:
No. It has little to do with what `browser` section in `package.json` does.
This line tells browserify to export `query` as `component-query` and it's needed since demo [index.html:39] requires `component-query`
Now that I think about it maybe I should just fix the demo... Thoughts?


[index.html:39]: https://github.com/component/popover/blob/master/test/index.html#L39